stories/covid-vax/covid_vax.ipynb

```diff
@@ -0,0 +1,1597 @@
+{
```

Comment:
<p>Does the legend for this plot need to be updated to show the fast and slow gov rate forecasts as dotted lines?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/alan-turing-institute/TuringDataStories/pull/138/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='74'/>

Reply:
<p>Also, wondering if there might be some value in having a visualisation that shows the cumulative totals as a % of the total adult population - this could mean simply changing the label on the y-axis, but I wonder if thinking about the % would lead to any other types of visualisations. Maybe it is better to keep the y-axis consistent to easily compare the plots, but I was thinking the % vaccinated (1st dose and 2nd dose) puts that 100% date in some context, in terms of where we are now...</p>

Reply:
<p>Ah I missed this one yesterday, I might see if I can get it to work with a second y-axis on the right-hand side showing the percentage.</p>